### DISCUSSION TITLE
Remove table of contents?

### DISCUSSION THREAD

**Discussion by axelboc** [axelboc/anki-ultimate-geography#420] [Ideas]
GitHub now provides table of contents for Markdown files automatically!

![image](https://user-images.githubusercontent.com/2936402/115112838-9db1ab00-9f87-11eb-9e93-3ef22d3034e7.png)

This makes the table of contents in the README and the CONTRIBUTING guide less relevant. Should we remove them?

It's nice having the table of contents directly visible, and GitHub's TOC is not as readable and scannable as ours, so not sure. Just thought I'd point out this new feature. 😊

**Comment by aplaice** [comment]
I also prefer our ToC.

It _might_ make sense to drop the ToCs if/when we'll be doing considerable changes to the structure of the files, to avoid having to worry about recreating/updating them (I often forget...). I don't think we should drop them proactively, though.

In any case, thanks for pointing out the cool feature, as it's useful for other READMEs on GitHub that don't have their own ToC!

<hr/>

(Also, I fixed a tiny typo directly on `master`. I hope that's OK! Opening a PR for one letter felt over-kill...)
